### DISCUSSION TITLE
Kobo (e-reader) client app

### DISCUSSION THREAD

**Discussion by Djiit** [go-shiori/shiori#545] [Feature requests]
Hey 👋

Is it possible to read bookmarked items from an eReader like a Kobo ? A bit like Pocket has a Kobo app that syncs items on the reader ?

Thanks

**Comment by fmartingr** [comment]
Hey @Djiit , I'm not familiar with this eReader to be honest. If it had a browser you could go to your shiori server and open your bookmakrs there, but I'm assuming this is not possible. Shiori doesn't have the capabilities to send articles to third-party applications or hardware.

**Comment by blogbourri** [comment]
Hello, I wanted to share this is the exact use case I'd most like to use Shiori for. To clarify, Kobo has a browser, but the way Shiori handles logging in breaks in the browser. If you disable javascript in firefox you can see what happens, the login button just spins perpetually and {{error}} appears at the top of the page. Basically there would need to be a "mobile" version of Shiori that doesn't use javascript for logging in and browsing. I'm assuming that is a non-trivial ask, but who knows. Doing some quick testing: 1) the current login does not work with javascript disabled 2) the current home page does not work with javascript disabled 3) the reader version of archived webpages DOES work with javascript disabled. So in theory, I should be able to read archived websites on a kobo ereader if i could just get to them.
